src/renderer/components/TimelineSpace/Contents/Home.vue

```diff
@@ -70,6 +72,43 @@ export default {
     },
     deleteToot (message) {
       this.$store.commit('TimelineSpace/Contents/Home/deleteToot', message)
+    },
+    async reload () {
+      const loading = this.$loading({
+        lock: true,
+        text: 'Loading',
+        spinner: 'el-icon-loading',
+        background: 'rgba(0, 0, 0, 0.7)'
+      })
+      const account = await this.$store.dispatch('TimelineSpace/localAccount', this.$route.params.id).catch(() => {
+        this.$message({
+          message: 'Could not find account',
+          type: 'error'
+        })
+      })
+      await this.$store.dispatch('TimelineSpace/stopUserStreaming')
+      await this.$store.dispatch('TimelineSpace/stopLocalStreaming')
+
+      await this.$store.dispatch('TimelineSpace/Contents/Home/fetchTimeline', account)
+        .catch(() => {
```

Comment:
I attempt to `catch` errors on only current timeline although, in the case of `initialize()`, catch multiple streamings errors at the same time.